sdk/src/main/scala/com/horizen/websocket/server/SidechainNodeChannelImpl.scala

```diff
@@ -156,13 +105,12 @@ class SidechainNodeChannelImpl() extends SidechainNodeChannel with ScorexLogging
     val sidechainNodeView: View = Await.result(viewAsync(), 5000 millis)
 
     val txids: util.ArrayList[String] = new util.ArrayList[String]()
-    val mempoolTxes: util.List[BoxTransaction[Proposition, Box[Proposition]]] = sidechainNodeView.pool.getTransactions
-    mempoolTxes.forEach(txs => txids.add(txs.id()))
+    sidechainNodeView.pool.take(200).foreach(txs => txids.add(txs.id()))
```

Comment:
is it fine to have a limit of `200`?
You may take `sidechainNodeView.pool.size` if you want to get all entries.